apps/verticalhome/js/pin_apps_default.json

```diff
@@ -0,0 +1,33 @@
+{"pin_apps_default":
+[
+  {"entry_point": "dialer",
+    "name": "Phone",
```

Comment:
You can't just hardcode english names like this. The homescreen needs to read a localized name for each app from the app's manifest file.

Localization is important. We've got to support it in this homescreen. If you can't fix this now, we need to at least file a followup bug to be sure you come back to this and do it properly.